ts/features/messages/components/MessageDetail/MessageDetailsAttachmentItem.tsx

```diff
@@ -9,28 +9,41 @@ import I18n from "../../../../i18n";
 type MessageDetailsAttachmentItemProps = {
   attachment: ThirdPartyAttachment;
   bottomSpacer?: boolean;
+  disabled?: boolean;
+  isPN?: boolean;
   messageId: UIMessageId;
+  onPreNavigate?: () => void;
   serviceId?: ServiceId;
 };
 
 export const MessageDetailsAttachmentItem = ({
   attachment,
   bottomSpacer,
+  disabled = false,
+  isPN = false,
   messageId,
+  onPreNavigate = undefined,
   serviceId
 }: MessageDetailsAttachmentItemProps) => {
   const { displayName, isFetching, onModuleAttachmentPress } =
-    useAttachmentDownload(messageId, attachment, false, serviceId);
+    useAttachmentDownload(
+      messageId,
+      attachment,
+      isPN,
+      serviceId,
+      onPreNavigate
+    );
   return (
     <>
       <ModuleAttachment
-        title={displayName}
-        isFetching={isFetching}
+        disabled={disabled}
         fetchingAccessibilityLabel={I18n.t(
           "features.messages.attachmentDownloadFeedback"
         )}
         format={"pdf"}
+        isFetching={isFetching}
         onPress={() => void onModuleAttachmentPress()}
```

Comment:
```suggestion
        onPress={onModuleAttachmentPress}
```
why define a new function?

Reply:
Because `onModuleAttachmentPress` is an async function. Using the `void` attribute when defining the new function allows the code to treat it as a non blocking call